### DISCUSSION TITLE
请问根据这个电子书教程这种证件检测，直接将多行放到一个框里，后续怎么做识别？

### DISCUSSION THREAD

**Discussion by kkpssr** [PaddlePaddle/PaddleOCR#13863] [General]
![image](https://github.com/user-attachments/assets/97613d4e-95ba-46cb-8c72-283c14994b4b)
如图，标注时将地址项都放到同一个检测框内，后续怎么进行识别呢，是还需要在检测加识别么

**Comment by UserWangZz** [comment]
目前识别都是基于单行的，可能这种情况你需要进一步的处理，或者是重新标注

**Reply by kkpssr** [reply]
> 目前识别都是基于单行的，可能这种情况你需要进一步的处理，或者是重新标注

您好，这是动手学ocr电子书里的内容，电子书里说的不是很清楚，我又看了看，请问是否检测的时候是多头检测，一方面输出单行文本检测，一方面输出文本块检测及标签。

**Reply by UserWangZz** [reply]
目前，PaddleOCR是两阶段，先通过文本检测划取文本区域，然后通过裁切后送入文本识别模型，这里这个可能只是一个单独的示意图，目前文本检测都是基于单行的。
